packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -585,6 +585,37 @@
           }
         }
       },
+      "observables": {
+        "properties": {
+          "createdAt": {
```

Comment:
Could not quite tell from look at the code. Are we intending to power search over all these fields? We should only add mappings for those we want to search for.